src/widgets/vitals/vital-record.component.tsx

```diff
@@ -6,6 +6,8 @@ import { createErrorHandler } from "@openmrs/esm-error-handling";
 import styles from "./vital-record.css";
 import SummaryCard from "../../ui-components/cards/summary-card.component";
 import dayjs from "dayjs";
+import VitalsForm from "./vitals-form.component";
+import { openWorkspaceTab } from "../shared-utils";
 
 export default function VitalRecord(props: VitalRecordProps) {
   const [vitalSigns, setVitalSigns] = useState(null);
```

Comment:
What's the type here?

Reply:
`VitalsForm` is a functional component. `openWorkspaceTab` is function with return type `void`

Reply:
I was referring to `useState`. What type is `vitalSigns`?

Reply:
Is this of type `PatientVitals`?